assembly/assembly-main/src/assembly/bin/che.sh

```diff
@@ -663,77 +360,25 @@ launch_docker_registry () {
     fi
 
     if ${CREATE_NEW_CONTAINER} ; then
-
       # Container in bad state or not found, kill and launch new container.
       kill_and_launch_docker_registry
-
     fi
 }
 
-launch_che_server () {
-
-  if [[ -n "${DOCKER_HOST}" ]]; then
-    strip_url ${DOCKER_HOST}
-  else
-    host=localhost
-  fi
-
-  if ! ${SKIP_PRINT_CLIENT}; then
-    print_client_connect
-  fi
-
-  if ${LAUNCH_REGISTRY} ; then
-    # Export the value of host here
-    # Will be used on Che properties to set the location of registry
-    export CHE_REGISTRY_HOST="${host}"
-    launch_docker_registry
+execute_che () { 
+  check_docker
+  determine_os
```

Comment:
`check_doker` and `determine_os` won't work when running in a non privileged container on Kubernetes or OpenShift